vms/platformvm/txs/builder/builder.go

```diff
@@ -864,33 +677,24 @@ func (b *builder) NewTransferSubnetOwnershipTx(
 	changeAddr ids.ShortID,
 	memo []byte,
 ) (*txs.Tx, error) {
-	ins, outs, _, signers, err := b.Spend(b.state, keys, 0, b.cfg.TxFee, changeAddr)
-	if err != nil {
-		return nil, fmt.Errorf("couldn't generate tx inputs/outputs: %w", err)
+	pBuilder, pSigner := b.builders(keys)
+
+	utils.Sort(ownerAddrs) // sort control addresses
```

Comment:
strictly speaking we were not sorting in `NewTransferSubnetOwnershipTx`. But we do it for `NewCreateSubnetTx, so we better be consistent